superduperdb/ext/transformers/training.py

```diff
@@ -607,8 +602,8 @@ def ray_train_func(train_loop_config):
         train_loop_args.build()
         return train_func(train_loop_args, train_dataset, eval_datasets, **kwargs)
 
-    if ray_address is not None:
-        ray.init(address=ray_address, ignore_reinit_error=True)
+    if not ray.is_initialized():
+        ray.init(ignore_reinit_error=True)
```

Comment:
will this init locally?

Reply:
If connected to a local cluster, it will connect to that Ray. If there is no Ray environment available, Ray will be started for multi-GPU training.